### PR TITLE
translate the solution from solution.cn.md to solution.en.md

### LINKED ISSUE #6070: [Translation] Translate technical articles concerning Apache ShardingSphere
Hi, community,

This issue is to call for volunteers to **translate some technical articles from Chinese into English**. Anyone who enjoys translation or wants to make some contributions to open source community is warmly welcomed! 👏 

### Competencies needed
1. English translation skill
2. Basically, know what ShardingSphere is doing

### Article list
- [x] 1. [Start ElasticJob off with new goal](https://mp.weixin.qq.com/s/XJFj_vOj3bR6HRQUzy1ikg) --> @rudy2steiner
- [x] 2. [How to construct the distributed database](https://mp.weixin.qq.com/s/YSoX0Apq6vQeu8TU_53EAw) --> @EasonFeng5870
- [x] 3. [Apache ShardingSphere is included in CNCF Landscape](https://mp.weixin.qq.com/s/hAxON1CMYvgFf6sFIHtDHA) --> @mlw12345678
- [x] 4. [The mixed open-source distributed transaction solution](https://mp.weixin.qq.com/s/fYESyuw7O4u7wB-tSWlhyg) --> @mlw12345678
- [x] 5. [Would you like to become an Apache committer](https://mp.weixin.qq.com/s/sFqOz10KTk23IhmmDTy67g) --> @mlw12345678
- [x] 6. [How automatic executor of ShardingSphere works](https://mp.weixin.qq.com/s/hIKZPJDfGfeduW4frwVN_w)
- [x] 7. [How to merge the child resultsets](https://mp.weixin.qq.com/s/5__SGXEdvGqkVhpnrW6zug)
- [ ] 8. [The quick explanation of ShardingSphere transaction module](https://mp.weixin.qq.com/s/0LtqRTL8HewUAdZCk8LRLg)
- [x] 9. [What is the ShardingProxy](https://mp.weixin.qq.com/s/bTX0n9tH4x8eue19b7rdGw) --> @wcsoft 

### How to submit your translation
1. Add your translation to `xxx.en.md` in [blog](https://github.com/apache/shardingsphere/tree/master/docs/blog/content/material).

2. Raise a PR for review.

### Notice
1. All the Chinese articles named `xxx.cn.md` could be found in the same directory of Your translation as well. 
2. If you want o find a partner to work together for these long articles, please be free to give `recruit comment` below. 😉 
3. You can give a brief translation for these author profiles in the articles. Like just translate their names and titles.

If you want to have a try, **please comment here with** `Article number and your estimated deadline`. 

Besides, **any feedback and comments are welcomed as well**! 😃

### PR DESCRIPTION
**Pull request by mlw12345678**
Fixes #6070 No4.

Changes proposed in this pull request:
-
-
-
image

### CODE REVIEW

**Comment by tristaZero** [comment]
Hi @mlw12345678 Glad to see you twice. :-)

It looks adequate at my quick look, but @kimmking will give a detailed review. Please give us some time. 

Cheers,
Trista

File: docs/blog/content/material/solution.en.md [hunk 1]
```diff
@@ -4,4 +4,252 @@ weight = 9
...
+
+Strictly guaranteeing ACID characteristics of transactions is a double-edged sword. In the process of transaction execution, all required resources need to be locked, so the XA protocol is more suitable for short transactions whose execution time is determined. For long transactions, the exclusive use of data during the entire transaction will cause the concurrency performance of business systems that rely on hot data to decline significantly. Therefore, in high-concurrency performance-oriented scenarios, distributed transactions based on the XA protocol two-phase commit type are not the best choice.
+
+#### Flexible transaction
+
+If the transaction that implements the ACID transaction element is called a rigid transaction, the transaction based on the BASE transaction element is called a flexible transaction. BASE is an abbreviation of the three elements of basic availability, flexible state and final consistency.
```
Comment (kimmking) [inline docs/blog/content/material/solution.en.md:62]: BASE is an abbreviation of the three elements of Basic Availability, Soft state and Eventually consistent.

File: docs/blog/content/material/solution.en.md [hunk 2]
```diff
@@ -4,4 +4,252 @@ weight = 9
...
+
+- Basically Available	Ensure that participants in distributed transactions are not necessarily online at the same time.
+
+- Soft state	It allows a certain delay in the system status update, and this delay may not be noticeable to customers.
+
+- Eventually consistent	Usually, the final consistency of the system is ensured by means of message passing.
```
Comment (kimmking) [inline docs/blog/content/material/solution.en.md:68]: Eventually consistent	Usually, the eventually consistency of the system is ensured by means of message passing.

File: docs/blog/content/material/solution.en.md [hunk 3]
```diff
@@ -4,4 +4,252 @@ weight = 9
...
+
+Saga is a typical flexible transaction manager. The concept of Sagas comes from a database paper more than thirty years ago [http://www.cs.cornell.edu/andru/cs711/2002fa/reading/sagas.pdf], a Saga transaction is A long-term transaction which is composed of multiple short-term transactions. In the distributed transaction scenario, we regard a Saga distributed transaction as a transaction composed of multiple local transactions, and each local transaction has a corresponding compensation transaction. During the execution of the Saga transaction, if an abnormality occurs in a certain step, the Saga transaction will be terminated, and the corresponding compensation transaction will be called to complete the related recovery operation, so as to ensure that the local transactions related to Saga are all executed successfully, or through compensation restore to the state before the transaction was executed.
```
Comment (kimmking) [inline docs/blog/content/material/solution.en.md:72]: a Saga transaction is a long-term

File: docs/blog/content/material/solution.en.md [hunk 4]
```diff
@@ -4,4 +4,252 @@ weight = 9
...
+
+The strong consistency transaction based on ACID and the final consistency transaction based on BASE are not silver bullets, and their greatest strengths can only be used in the most suitable scenarios. The following table can be used to compare the differences between them in detail to help developers choose technologies.
+
+<center>
+
+
+|        Contrast         |             Local Transaction             |        Two-phase Submission         |        Flexible Transaction         |
```
Comment (kimmking) [inline docs/blog/content/material/solution.en.md:81]: Two-Phase Commit

**Pull request merged**